docs/sources/flow/reference/components/otelcol.receiver.otlp.md

```diff
@@ -168,6 +170,24 @@ request. The following headers are always implicitly allowed:
 
 If `allowed_headers` includes `"*"`, all headers will be permitted.
 
+### output block
+
+The `output` block configures a set of components to send received telemetry
+data to.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`metrics` | `list(otelcol.Consumer)` | List of consumers to send metrics to. | `[]` | no
+`logs` | `list(otelcol.Consumer)` | List of consumers to send logs to. | `[]` | no
+`traces` | `list(otelcol.Consumer)` | List of consumers to send traces to. | `[]` | no
+
+The `output` block must be specified, but all of its arguments are optional. By
+default, telemetry data will be dropped. To send telemetry data to other
+components, configure the `metrics`, `logs`, and `traces` arguments
+accordingly.
+
```

Comment:
Is it OK that we don't have examples for otelcol.processor.batch and otelcol.processor.batch?

Reply:
Yeah, I'm planning on adding examples after #2288 is implemented (since that would allow examples for an entire pipeline of OpenTelemetry Collector components)